### PR TITLE
backport to release/v1.0 (docs on 'dump' command)

### PR DESCRIPTION
**Pull request by issmirnov**
Clone of #40 - let's keep discussion over there.

### CODE REVIEW

File: content/doc/usage/backup-and-restore.en-us.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  sidebar:
+    parent: "usage"
+    name: "Backup and Restore"
+    weight: 11
+    identifier: "backup-and-restore"
+---
+
+# Backup and Restore
+
+Gitea currently has a `dump` command that will save your installation to a zip
```
Comment (tboerger) [inline content/doc/usage/backup-and-restore.en-us.md]: We have written all docs without manual line breaks within the paragraphs.

Comment (lunny) [reply content/doc/usage/backup-and-restore.en-us.md]: ^

Comment (tboerger) [reply content/doc/usage/backup-and-restore.en-us.md]: Same applies for the master branch of course.

**Comment by issmirnov** [comment]
Updated PR with same commit as in #51 - line breaks removed and SQL comment clarified.

**Comment by lunny** [comment]
LGTM

**Comment by appleboy** [comment]
LGTM

**Comment by bkcsoft** [comment]
LGTM

**Comment by bkcsoft** [comment]
make LGTM work

**Pull request merged**